tests/tests.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+
+@pytest.mark.django_db
```

Comment:
Why you need @pytest.mark.django_db ?
If you dont need then remove import pytest also.

Reply:
You're right ill remove that